utils/server-instance.js

```diff
@@ -31,6 +31,8 @@ class ServerInstance {
     this._app = express();
     this._app.use(cookieParser());
 
+    this._counter = 1;
```

Comment:
I'm not clear on the benefit of making that change.

With a counter, you're guaranteed that each response will be different, eliminating the infinitesimal chance that two random numbers will be the same, and the slightly more realistic issue of two requests in quick succession returning the same `Date.now()`.

Re: putting in `/nocache/` in the URL, there are some endpoints that respond with a `max-age` of 1 day, and those don't have `/1day/` in the URL.

If you want to change the URL to reflect the idea that its going to return a different value each time a page makes a request, what if I renamed it to `/__echo/unique`?